pkg/pillar/cmd/client/client.go

```diff
@@ -230,7 +230,7 @@ func Run() { //nolint:gocyclo
 		DeviceNetworkStatus: clientCtx.deviceNetworkStatus,
 		FailureFunc:         zedcloud.ZedCloudFailure,
 		SuccessFunc:         zedcloud.ZedCloudSuccess,
-		NetworkSendTimeout:  clientCtx.globalConfig.NetworkSendTimeout,
+		NetworkSendTimeout:  types.GetGlobalValue(types.NetworkSendTimeout, clientCtx.configItemValueMap).IntValue,
```

Comment:
this should be a method of clientCtx.configItemValueMap

clientCtx.configItemValueMap.ItemValue(<key>)